Story/OnlineUIComponents/PlayerSpecificOnlineHud.cs

```diff
@@ -92,16 +94,19 @@ public override void Update()
                 }
             }
 
-            if (camera.room == null || !camera.room.shortCutsReady) return;
             if (!clientSettings.inGame) return;
+            if (camera.room == null || !camera.room.shortCutsReady) return;
             if (abstractPlayer == null)
             {
                 RainMeadow.Debug("finding player abscrt for " + clientSettings.owner);
                 if (clientSettings.avatarId.FindEntity(true) is OnlineCreature oc)
                 {
                     abstractPlayer = oc.abstractCreature;
                 }
-                return;
+                else
```

Comment:
redundant else because either way we will return if abs player is not found, but not something to lose sleep over.

Reply:
doesn't it set the abstractplayer on success?

Reply:
yeah and abstractPlayer == null won't run again

Reply:
eh we save one more call to Update, imma leave it in